src/components/safe-apps/SafeAppsSection/CollapsibleSection.tsx

```diff
@@ -0,0 +1,81 @@
+import Accordion from '@mui/material/Accordion'
+import AccordionSummary from '@mui/material/AccordionSummary'
+import AccordionDetails from '@mui/material/AccordionDetails'
+import Typography from '@mui/material/Typography'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import Grid from '@mui/material/Grid'
+import { AppCard } from '@/components/safe-apps/AppCard'
+import { SectionProps } from './types'
+import { AddCustomAppCard } from '@/components/safe-apps/AddCustomAppCard'
+
+const CollapsibleSection = ({
+  title,
+  apps,
+  onPinApp,
+  prependAddCustomAppCard,
+  onAddCustomApp,
+  pinnedIds,
+  cardVariant,
+}: SectionProps) => {
+  const columnSpacing = cardVariant === 'compact' ? 3 : 2
+
+  return (
+    <Accordion
+      sx={{
+        boxShadow: 'none',
+        border: 'none',
+        background: 'transparent',
+      }}
+      defaultExpanded
+    >
+      <AccordionSummary
+        expandIcon={<ExpandMoreIcon />}
+        sx={({ spacing }) => ({
+          padding: `0 ${spacing(3)}`,
+          justifyContent: 'flex-start',
+          background: 'transparent',
+          '& > .MuiAccordionSummary-content': {
+            flexGrow: 0,
+            '&.Mui-expanded': {
+              margin: `${spacing(2)} 0`,
+            },
+          },
+          '&.Mui-expanded': {
+            minHeight: 48,
+          },
+        })}
+      >
+        <Typography variant="caption" fontWeight={700} sx={({ palette }) => ({ color: palette.secondary.light })}>
```

Comment:
@iamacook @usame-algan do we have an MUI preset for this style?

Reply:
`"caption"` is a variant [set in our theme already](https://github.com/safe-global/web-core/blob/main/src/styles/theme.ts#L94-L97).

Reply:
I meant for the secondary.light color.

Reply:
[Yes](https://github.com/safe-global/web-core/blob/main/src/styles/colors.ts#L15).

Reply:
So is there something else I should use? :D

Reply:
Never mind Mikhail, I thought we had like a preset for size and color.